bin/mark_changepoints_in_json

```diff
@@ -173,24 +173,35 @@ class Segments(object):
             if math.fabs(self.segments[0].mean - self.segments[1].mean) < self.delta:
                 self._merge_segment(0)
 
+    def _is_segment_a_slowdown(self, index, pc_bound=0.1):
+        """Test whether mean of segment index is slower than the last segment
+        mean by >= pc_bound percent.
+        """
+        if self.segments[index].mean <= self.segments[-1].mean:
+            return False  # Segment index is faster than segment -1.
+        pc = math.fabs(self.segments[index].mean - self.segments[-1].mean /
+                       self.segments[-1].mean * 100.0)
+        return pc >= pc_bound
+
     def get_classification(self):
         """Return the classification for this run sequence.
         """
+        PC_BOUND = 0.01  # TODO: Factor out as a CLI option before merging with master.
         classification = ''
         # Flat -> no changepoints.
         if len(self.segments) == 1:
```

Comment:
Maybe we can start with a variant of the algorithm I outlined? The thing that it doesn't do -- and I don't even totally know *what* it should do -- is deal with no steady state. But that's something we can probably worry about as a second step.